homeassistant/core.py

```diff
@@ -785,8 +788,12 @@ def async_add_import_executor_job(
     ) -> asyncio.Future[_T]:
         """Add an import executor job from within the event loop."""
         task = self.loop.run_in_executor(self.import_executor, target, *args)
-        self._tasks.add(task)
-        task.add_done_callback(self._tasks.remove)
+
+        tracked = asyncio.current_task() in self._tasks
+        task_bucket = self._tasks if tracked else self._background_tasks
+        task_bucket.add(task)
+        task.add_done_callback(task_bucket.remove)
+
```

Comment:
I'm not sure import executor jobs need to even be tracked

Reply:
Actually we have no fire and forgets ones of these in the code base so it's safe to remove

Reply:
https://github.com/home-assistant/core/pull/114453